poprox-db/migrations/versions/2024_07_26_1352-98ff1c3ebe33_add_web_login_table.py

```diff
@@ -0,0 +1,65 @@
+"""add web_login table
+
+Revision ID: 98ff1c3ebe33
+Revises: 211c6b3ac5de
+Create Date: 2024-07-26 13:52:10.169410
+
+"""
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects.postgresql import JSONB
+
+# revision identifiers, used by Alembic.
+revision: str = "98ff1c3ebe33"
+down_revision: Union[str, None] = "211c6b3ac5de"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    op.create_table(
+        "web_logins",
+        sa.Column(
+            "web_login_id",
+            sa.UUID,
+            primary_key=True,
+            server_default=sa.text("gen_random_uuid()"),
+        ),
+        sa.Column("account_id", sa.UUID, nullable=False),
+        sa.Column("newsletter_id", sa.UUID, nullable=True),
```

Comment:
Does this imply that each individual newsletter has a distinct login for the same user?